### PR TITLE
Homework6: Mikita Shnipau

### PR DESCRIPTION
**Pull request by shnipa**

### CODE REVIEW

File: tests/test_pull.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        """Init"""
+
+    def test_get_git_json(self):
+        """Test git key"""
+        self.assertTrue(pulls.get_git_json())
+
+    def test_get_pulls(self):
+        """"Test states"""
+        result = pulls.get_pulls("accepted")
+        self.assertIsInstance(result, list)
```
Comment (alenaPy) [inline tests/test_pull.py:18]: Actually, coverage is good, but tests don't verify anything except of return type.

File: tests/test_shnipov.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+json = [
```
Comment (alenaPy) [inline tests/test_shnipov.py:1]: Please add `__init__.py` to tests\ directory. Otherwise, nose doesn't find your tests

**Comment by alenaPy** [comment]
nosetests --with-coverage --cover-package=handlers/ --cover-erase tests/test_pull.py 
E..
======================================================================
ERROR: Test get all data
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/auser/projects/devops/devops_lab/tests/test_pull.py", line 28, in test_get_data_all
    self.assertEqual(pulls.get_data_all(shnipov),
  File "/home/auser/projects/devops/devops_lab/handlers/pulls.py", line 48, in get_data_all
    for i in res:
TypeError: 'module' object is not iterable
```
```

File: tests/test_pull.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    def setUp(self):
+        """Init"""
+
+    def test_get_git_json(self):
+        """Test git key"""
+        self.assertTrue(pulls.get_git_json())
+
+    def test_get_pulls(self):
+        """"Test states"""
+        result = pulls.get_pulls("accepted")
```
Comment (alenaPy) [inline tests/test_pull.py:17]: You shouldn't make tests for functions with external requests.

**Comment by shnipa** [comment]
accidentally added a key

**Pull request closed without merging**